src/app/pcpolicy/pcpolicy.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import { DashboardService } from '../services/dashboard.service';
+
+@Component({
+  selector: 'app-pcpolicy',
+  templateUrl: './pcpolicy.component.html',
+  styleUrls: ['./pcpolicy.component.scss']
+})
+export class PcpolicyComponent implements OnInit {
+  email: String;
+  public position= 'col-md-10 col-md-offset-2 pcpolicybody';
+  constructor(private _dashboardService: DashboardService) { }
+
+  ngOnInit() {
+    this._dashboardService.getUserInfo().subscribe(response => {
+        this.email = response.user.email;
+    });
+  }
+
+  toggle() {
+    if (this.position === 'col-md-10 col-md-offset-2 pcpolicybody text-center') {
```

Comment:
1. `pcpolicybody text-center` are fixed classes, not necessary here.
2. thought of ternary operator?